src/devices/Lis3Dh/Lis3DhI2c.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Device.Model;
+using System.Numerics;
+
+namespace Iot.Device.Lis3DhAccelerometer
+{
+    internal sealed class Lis3DhI2c : Lis3Dh
+    {
+        private I2cDevice _i2c;
+
+        public Lis3DhI2c(I2cDevice i2cDevice)
+        {
+            _i2c = i2cDevice ?? throw new ArgumentNullException(nameof(i2cDevice));
+        }
+
+        /// <inheritdoc/>
+        public override void Dispose()
+        {
+            _i2c?.Dispose();
```

Comment:
don't we do that in other bindings? I've copy & pasted the template for this binding including this specific line of code. I2cDevice is not much reusable after that

Reply:
https://github.com/dotnet/iot/blob/78a7365d3b7a0061a7c9d4ed38d644a3cc98dabf/src/devices/Lps25h/Lps25h.cs#L113
https://github.com/dotnet/iot/blob/78a7365d3b7a0061a7c9d4ed38d644a3cc98dabf/src/devices/Mpu6886/Mpu6886.cs#L184
https://github.com/dotnet/iot/blob/68e50c9712c07c17ba6058ae28444ed3cddfa4b0/src/devices/Mfrc522/Mfrc522.cs#L839
https://github.com/dotnet/iot/blob/e0dbd643875189804da3f090fd26227f9b01a13c/src/devices/Adxl357/Adxl357.cs#L187

4 random I2C devices I opened

Reply:
well, we should not because we don't create those!

Reply:
I believe intent is that you do - the device takes ownership of that object since it's useless without the device binding - it's bound to a specific address (as oposed to I2cBus i.e.)

Reply:
Still. And we have to then make things clear about this.